docs/Testing.md

```diff
@@ -0,0 +1,70 @@
+# Testing for Infrastructure Tools
+
+This document will include all of the ways in which testing is done in the infrastructure repository. 
+
+## Vagrant Playbook Check
+
+The Vagrant Playbook Check (VPC) job is used to test changes made to the [windows and linux playbooks](https://github.com/adoptium/infrastructure/tree/master/ansible/playbooks).
+
+The job is built in our [Jenkins server](https://ci.adoptium.net/job/VagrantPlaybookCheck/). The job runs a user's pull request onto the supported operating systems using Vagrant virtual machines. The vagrant files for the supported operating systems can be found [here](https://github.com/adoptium/infrastructure/tree/master/ansible/vagrant).
+
+Currently, the supported x86-64 systems are:
+- Windows 2012 and 2022
+- Debian 8 and 10
+- Ubuntu 16.04, 18.04, 20.04 and 22.04
+- CentOS 6, 7 and 8
+- Solaris 10
+
+## Qemu Playbook Check
+
+The QEMU Playbook Check (QPC) job is similar to the VPC job except it is not limited to x86-64 virtual machines.
+
+It is built in our [Jenkins server](https://ci.adoptium.net/job/QEMUPlaybookCheck/). The job itself runs [this script](https://github.com/adoptium/infrastructure/blob/master/ansible/pbTestScripts/qemuPlaybookCheck.sh).
+
+Currently, the supported platorms are:
+- Ubuntu 18.04 on ppc64le, aarch64 and s390x
+- Debian 11 on risc-v
```

Comment:
Suggested change to make it more explicitl, like the other ones (There is a 32-bit RISC-V standard too but that's not what we're using here)
```suggestion
- Debian 11 on riscv64
```